src/containers/register/index.tsx

```
@@ -141,6 +143,28 @@ export default function RegisterContainer() {
+				query: AuthGQL.QUERIES.GET_SOCIAL_URL,
+				variables: { provider },
+				fetchPolicy: 'no-cache'
+			});
+
+			if (data?.getOAuth2Link) {
+				window.location.href = data.getOAuth2Link;
+			} else {
+				alertStore.error('소셜 로그인 URL을 가져올 수 없습니다.');
+				sessionStorage.removeItem('oauth_provider');
+			}
+		} catch (error: any) {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)